ui/components/multichain/address-list-item/address-list-item.tsx

```diff
@@ -67,17 +76,43 @@ export const AddressListItem = ({
       className="address-list-item"
       alignItems={AlignItems.center}
     >
-      <AvatarAccount
-        borderColor={BorderColor.transparent}
-        size={AvatarAccountSize.Md}
-        address={address}
-        variant={
-          useBlockie
-            ? AvatarAccountVariant.Blockies
-            : AvatarAccountVariant.Jazzicon
-        }
-        marginInlineEnd={2}
-      />
+      {process.env.REMOVE_GNS ? (
+        <BadgeWrapper
+          badge={
+            <AvatarNetwork
+              size={AvatarNetworkSize.Xs}
+              name={allNetworks?.[chainId as Hex]?.name}
+              src={getImageForChainId(chainId) || undefined}
```

Comment:
I think `getImageForChainId` returns `undefined` when an item isn't found, so we don't need `undefined`

Reply:
Nice catch, updated here https://github.com/MetaMask/metamask-extension/pull/31488/commits/60b31804f63afc480c5d02c920b48d817f6f211b